tracer/src/Datadog.Trace/Tagging/TraceTagCollection.cs

```diff
@@ -32,7 +29,7 @@ public TraceTagCollection(List<KeyValuePair<string, string>>? tags = null, strin
         /// </summary>
         public int Count => _tags?.Count ?? 0;
 
-        public void SetTag(string name, string? value)
+        public void SetTag(string name, string? value, bool replaceIfExists = true)
```

Comment:
don't...mention...the default..values.... :eyetwitch:

Reply:
Yeah, I wasn't a big fan either, but it made things easier at the moment. I now split this into methods, `TryAddTag()` and `SetTag()` (which internally call this one, but the parameter is no longer optional). Better? Or did you just want every call to pass the boolean explicitely?

Reply:
> which internally call this one, but the parameter is no longer optional

Sounds ideal to me 🙂